homeassistant/components/switcher_kis/sensor.py

```diff
@@ -0,0 +1,251 @@
+"""Home Assistant Switcher Component.
+
+For controlling the Switcher Boiler Device (https://www.switcher.co.il/).
+Please follow configuring instructions here:
+    https://www.home-assistant.io/components/switcher_kis/
+
+Author: Tomer Figenblat
+
+This cannot be configured as a sensor platform,
+Please follow the instruction of configuring the switcher_kis component.
+"""
+
+from asyncio import TimeoutError as AsyncioTimeoutError
+from asyncio import wait_for
```

Comment:
Yeah I think I'm done using `isort`, that (and much more) are its doings.
I kept combining them in one line and `isort` kept splitting them up.
Finally I caved, because the [documentation recommended it](https://developers.home-assistant.io/docs/en/development_guidelines.html#ordering-of-imports).
But I think from now on I will arrange the imports manually.